docs/remote-runners.md

```diff
@@ -6,7 +6,7 @@ Earthly supports running builds remotely via remote runners. Remote runners allo
 
 Typical use cases for remote runners include:
 
-* Speeding up CI builds in sandboxed CI environments such as GitHub Actions, GitLab, CircleCI, and others. Most CI build times are improved by a factor of 2-4X via Satellites.
+* Speeding up CI builds in sandboxed CI environments such as GitHub Actions, GitLab, CircleCI, and others. Most CI build times are improved by a factor of 2-20X via Satellites.
 * Executing builds on AMD64/Intel architecture natively when working from an Apple Silicon machine (Apple M1/M2).
```

Comment:
This works both ways, right? You can execute ARM64 builds from AMD64 architectures too, right? Maybe change to "Executing builds on AMD64/Intel architecture natively when working from an Apple Silicon machine (Apple M1/M2) and vice versa."

Reply:
Yes - fixed!